routes/originMavenGoogle.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const asyncMiddleware = require('../middleware/asyncMiddleware')
+const router = require('express').Router()
+const requestPromise = require('request-promise-native')
+const { promisify } = require('util')
+const parseString = promisify(require('xml2js').parseString)
+const { uniq } = require('lodash')
+
+// Get versions
+router.get(
```

Comment:
One more suggestion on this file - most of the routes for origins have a route similar to this one that allows for search similar to [this one](https://github.com/clearlydefined/service/blob/87181ac6d06a2029ed8d402bdf8731eef686e1d2/routes/originMaven.js#L29). I would add that route in, but have it return an error to the user indicating that search is not available for the source and to please request a specific revision. This will make it consistent with the other origins.

Reply:
Added search route and returning error message indicating search is not supported